care/users/api/viewsets/change_password.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework import serializers
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework.generics import UpdateAPIView
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+
+User = get_user_model()
+
+class ChangePasswordSerializer(serializers.Serializer):
+    model = User
+
+    """
+    Serializer for password change endpoint.
+    """
+    old_password = serializers.CharField(required=True)
```

Comment:
Why is this field not `write_only=True`? ideally, both these fields do not need `write_only` tags since they are not used to render any content.